openlibrary/plugins/books/tests/test_readlinks.py

```diff
@@ -0,0 +1,105 @@
+from openlibrary.plugins.books import readlinks
+import pytest
```

Comment:
```suggestion
```

Reply:
Checking the code here, I think the same.